.github/CODEOWNERS

```diff
@@ -258,10 +258,10 @@
 /comp/core/sysprobeconfig @DataDog/ebpf-platform
 /comp/core/tagger @DataDog/container-integrations
 /comp/core/workloadmeta @DataDog/container-integrations
-/comp/forwarder/eventplatform @DataDog/agent-metrics-logs
-/comp/forwarder/eventplatformreceiver @DataDog/agent-metrics-logs
-/comp/forwarder/orchestrator @DataDog/agent-metrics-logs
-/comp/forwarder/orchestrator/orchestratorinterface @DataDog/agent-metrics-logs
+/comp/forwarder/eventplatform @DataDog/agent-processing-and-routing
+/comp/forwarder/eventplatformreceiver @DataDog/agent-processing-and-routing
+/comp/forwarder/orchestrator @DataDog/agent-processing-and-routing
+/comp/forwarder/orchestrator/orchestratorinterface @DataDog/agent-processing-and-routing
```

Comment:
I think these are redundant due to `/comp/forwarder` above.

Reply:
You're right! Fixing this, thank you